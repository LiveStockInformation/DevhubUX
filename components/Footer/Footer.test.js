import * as React from 'react'
import { render, cleanup } from '@testing-library/react'
import '@testing-library/jest-dom'
import { Footer } from '.'

afterEach(cleanup)

describe('Footer Component', () => {
  it('renders without crashing', () => {
    const { asFragment } = render(<Footer />)
    expect(asFragment()).toMatchSnapshot()
  })
})
