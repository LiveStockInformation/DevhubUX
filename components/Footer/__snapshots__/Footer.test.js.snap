// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component renders without crashing 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <div
      class="footer__content"
    >
      <p
        class="ls-visually-hidden"
      >
        Footer links
      </p>
      <ul
        class="footer__inline-list"
      >
        <li
          class="footer__inline-list-item"
        >
          <a
            class="ls-link footer__link"
            href="/cookies"
          >
            Cookies
          </a>
        </li>
        <li
          class="footer__inline-list-item"
        >
          <a
            class="ls-link footer__link"
            href="/privacy-policy"
          >
            Privacy policy
          </a>
        </li>
        <li
          class="footer__inline-list-item"
        >
          <a
            class="ls-link footer__link"
            href="/terms-and-conditions"
          >
            Terms and conditions
          </a>
        </li>
        <li
          class="footer__inline-list-item"
        >
          <a
            class="ls-link footer__link"
            href="/accessibility-statement"
          >
            Accessibility statement
          </a>
        </li>
      </ul>
    </div>
  </footer>
</DocumentFragment>
`;
